rr/gstreamer/gst_media.py

```diff
@@ -179,6 +181,9 @@ def get_data(self):
     def get_sample(self):
         return self.sample
 
+    def get_gst_media_obj(self):
```

Comment:
the interface should be generic, not gstreamer oriented. Rename this method as `get_media`